schedule/yast/yast2_gui/yast2_gui_sle.yaml

```diff
@@ -73,6 +74,19 @@ conditional_schedule:
         - yast2_gui/yast2_datetime
         - yast2_gui/yast2_hostnames
 test_data:
+  mistakes:
+    - mistake: typo_in_property
+      make_typo: "sed -i 's/dhcp/dgcp/' /etc/sysconfig/network/ifcfg-eth0"
+      fix_typo: "sed -i 's/dgcp/dhcp/' /etc/sysconfig/network/ifcfg-eth0"
```

Comment:
I wonder if we would better just backup file in the test module and copy it in order to get to the same state before next test. It would simplify reverting part and won't need special command for each typo.

Reply:
Makes sense.

Reply:
So I did that.